source/documentation/common/security/proc-Applying_the_DISA_STIG_Security_Profile_RHV.adoc

```diff
@@ -6,9 +6,12 @@
 :_content-type: PROCEDURE
 [id='applying-the-disa-stig-profile-rhvh_{context}']
 
-= Applying the DISA STIG profile in Red Hat Virtualization Host
+= Applying the DISA STIG profile in RHEL based hosts and the standalone Manager
```

Comment:
but it can also be set in HE, so why saying "standalone"?

Reply:
Because the HE is covered down below.